main.go

```diff
@@ -102,6 +103,7 @@ var modules = map[string](func(*powerline)){
 	"time":     segmentTime,
 	"user":     segmentUser,
 	"venv":     segmentVirtualEnv,
+	"set-term-title": segmentSetTermTitle,
```

Comment:
This list is supposed to be alphabetically sorted.

Reply:
Also, you’ll need to add it to the list at https://github.com/mexchip/powerline-go/blob/3b0da7c7667ed39ff9d4244db465d5a6271bf0a3/main.go#L163